data-governance/no-code-access.mdx

```diff
@@ -1,51 +1,54 @@
 ---
 title: "Compliance and Security"
+description: "All about Octomind data access and governance"
 icon: "code"
 ---
 
-## CI/CD Integration
+## No access to source code needed
 
-Our GitHub and Azure DevOps integrations are bi-directional. They trigger test runs and comment results back into your pull requests.
-These only require comment permissions. Please check out the respective section ([GitHub](/integrations/quickstart-GitHub),[Azure DevOps](/integrations/quickstart-Azure)) in our documentation for more details.
+We operate by simulating a real user interacting with your web app, using your app's **DOM** to identify actions like clicks or text
+inputs. We are also using **screenshots** of your web app. This doesn't require access to your source code. We are only seeing the parts of your app
+which are exposed to your users.
 
-<Tip>Both tools are open source so that you can run an audit.</Tip>
+## Data collected when using the Octomind application
 
-Other intergrations into Vercel, Jenkins or the curl command are not able to comment back. Hence they do not need permissions.
+The Octomind app is **not** collecting user or app specific data. Octomind is **not** using any customer data for training purposes. DOM structure and screenshots gathered by the AI agent to examine user flows
+within your app are not stored on our computers. They are only temporarily used to identify the next best action to take from a users perspective or to propose test cases.
 
-## Data gathered from using the application
+Data from test runs like traces and screenshots are **deleted after 15 days automatically** as part of our data retention policy. From that time onwards you cannot access traces and screenshots anymore.
 
-Octomind is not collecting user or app specific data. Octomind is not using any customer data for training purposes. Dom structure and screenshots gathered by the AI agent to examine user flows
-within your app are not stored on our computers. They are only temporarely used to identify the next best action to take from a users perspective or to propose test cases. Data from test runs like
-traces and screenshots are deleted after 15 days automatically as part of our data retention policy. From that time onwards you cannot access traces and screenshots anymore.
+## CI/CD Integration
+
+Our **GitHub** and **Azure DevOps** integrations are bi-directional. They trigger test runs and comment results back into your pull requests.
+They require comment permissions only. Please check out the respective section ([GitHub](/integrations/quickstart-GitHub),[Azure DevOps](/integrations/quickstart-Azure)) in our documentation for more details.
+
+<Tip>Both tools are open source so that you can run an audit.</Tip>
```

Comment:
I would not call these integrations "tools". Also not sure about "open source". When saying open source I would expect a public repository, but in this case it is simply publicly documented, isn't it? Maybe I#m wrong and you meant something else

Reply:
ah, i didn't rewrite this one. It is a public repo indeed. I'll change the 'tool'.